db/crud.go

```diff
@@ -1106,13 +1106,25 @@ func (db *DatabaseCollectionWithUser) PutExistingCurrentVersion(ctx context.Cont
 			if addNewerVersionsErr != nil {
 				return nil, nil, false, nil, addNewerVersionsErr
 			}
+			if docHLV.MergeVersions != nil {
+				if doc.HLV.MergeVersions == nil {
+					doc.HLV.MergeVersions = make(map[string]uint64)
+				}
+				doc.HLV.MergeVersions = docHLV.MergeVersions
```

Comment:
Why isn't `doc.HLV` initialized with this set if we always do it, or
why doesn't `doc.HLV.AddNewerVersions(docHLV)` do this (if `MergeVersions` are not considered newer that'd be a good reason)

Reply:
My thinking was that they aren't necessarily newer versions. I have stopped the repeated code as you pointed out in comment above (moved outside the if statement)